packages/larva-patterns/components/c-figure/c-figure.twig

```diff
@@ -0,0 +1,64 @@
+{% if c_figure_permalink_url %}
+	<a href="{{ c_figure_permalink_url }}" class="c-figure__link {{ c_figure_link_classes }}"
```

Comment:
`c-figure` is a legacy version of `o-figure` - the figure pattern should be an "object" since it includes other patterns. Are there updates that can be made to `o-figure` instead, or can the consuming theme be updated to use `o-figure`?